text/morph.js

```diff
@@ -188,6 +188,9 @@ export class Text extends Morph {
       },
 
       padding: {
+        spec: {
```

Comment:
That's a good question. I have to admit I first just put this new set of information into a separate field to simplify the implementation process, but yeah we could also just add new fields to the properties. My hunch was probably that we may wanna create type objects in the future that provide more rich interaction capabilities and these are easier to extract if all this info is already stored in a separate field.